src/rez/package_cache.py

```diff
@@ -116,7 +129,7 @@ def get_cached_root(self, variant):
 
         return rootpath
 
-    def add_variant(self, variant, force=False):
+    def add_variant(self, variant, force=False, wait_for_copying=False, logger=None):
```

Comment:
I think `wait_for_copying` should be True by default.

Reply:
That would change the default behavior from async to sync, which I don't think we want to do, do we?